docs/src/pages/components/slider/MinimumDistanceSlider.js

```diff
@@ -46,25 +45,19 @@ export default function MinimumDistanceSlider() {
 
   return (
     <Box sx={{ width: 300 }}>
-      <Typography id="minimum-distance-demo" gutterBottom>
-        Minimum distance
-      </Typography>
       <Slider
+        getAriaLabel={() => 'Minimum distance'}
```

Comment:
```suggestion
        aria-label="Minimum distance"
```
?

Reply:
Let's use one way for adding the aria label.

Reply:
I had assumed aria-label for single value and getAriaLabel for multiple values, as each thumb needs its label.

Reply:
Ah got ya! Wasn't clear why we use different methods for adding it.